### PR TITLE
Feature/new methods

### PR DESCRIPTION
**Pull request by andrey-semin**
Added a couple of new methods (#28)

### CODE REVIEW

File: src/Smithers.js [hunk 1]
```diff
@@ -22,4 +22,34 @@ export default class Smithers {
...
+        return this.caller.get(requestUrl, config);
+    }
+
+    specificBuild(name, build, config) {
+        const requestUrl = createPath(paths.specificBuild, { name, build });
+        return this.caller.get(requestUrl, config);
+    }
+
+    overallLoad(config) {
+        const requestUrl = createPath(paths.overallLoad);
```
Comment (clementallen) [inline src/Smithers.js]: No need to put `paths.overallLoad` through the createPath method as no variable injection required.

File: src/Smithers.js [hunk 2]
```diff
@@ -22,4 +22,34 @@ export default class Smithers {
...
+        return this.caller.get(requestUrl, config);
+    }
+
+    queue(config) {
+        const requestUrl = createPath(paths.queue);
```
Comment (clementallen) [inline src/Smithers.js]: Same as comment on `overallLoad` method

File: src/Smithers.js [hunk 3]
```diff
@@ -22,4 +22,34 @@ export default class Smithers {
...
+        const requestUrl = createPath(paths.lastStableBuild, { name });
+        return this.caller.get(requestUrl, config);
+    }
+
+    lastUnsuccessfulBuild(name, config) {
+        const requestUrl = createPath(paths.lastUnsuccessfulBuild, { name });
+        return this.caller.get(requestUrl, config);
+    }
+
+    specificBuild(name, build, config) {
```
Comment (clementallen) [inline src/Smithers.js]: `build` looks better as `buildNumber` like you've put in the README

Comment (andrey-semin) [reply src/Smithers.js]: For me `buildNumber` looks more verbose here

Comment (clementallen) [reply src/Smithers.js]: Sorry, could have been clearer!

**Comment by clementallen** [comment]
Thanks @andrey-semin 👍

**Pull request merged**